src/layers/mlp.jl

```diff
@@ -1,16 +1,47 @@
 """
-    mlp_block(planes, hidden_planes; dropout = 0., dense = Dense, activation = gelu)
+		mlp_block(inplanes, hidden_planes; outplanes = inplanes, dropout = 0., activation = gelu)
 
-Feedforward block used in many vision transformer-like models.
+Feedforward block used in many MLPMixer-like and vision-transformer models.
 
 # Arguments
-- `planes`: Number of dimensions in the input and output.
+- `inplanes`: Number of dimensions in the input.
 - `hidden_planes`: Number of dimensions in the intermediate layer.
+- `outplanes`: Number of dimensions in the output - by default it is the same as `inplanes`.
 - `dropout`: Dropout rate.
-- `dense`: Type of dense layer to use in the feedforward block.
 - `activation`: Activation function to use.
 """
-function mlp_block(planes, hidden_planes; dropout = 0., dense = Dense, activation = gelu)
-  Chain(dense(planes, hidden_planes, activation), Dropout(dropout),
-        dense(hidden_planes, planes), Dropout(dropout))
+function mlp_block(inplanes, hidden_planes; outplanes = inplanes, dropout = 0., activation = gelu)
```

Comment:
Should `outplanes` be positional?

Reply:
Sure we can keep it as is. I just thought that it should be for consistency since all the other dimension arguments are.